mm_cataloguer/generate_manifest.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from modelmeta import DataFile, DataFileVariable, Ensemble, EnsembleDataFileVariables
+from datetime import datetime
+
+def list_filepaths(session, ensemble, since):
+    if ensemble == ["all"]:
+        ensemble = ["all_files"]
+    since = datetime.strptime(since, "%Y-%m-%d")
+    print(session.query(DataFile.filename).join(DataFileVariable)
+                                          .join(EnsembleDataFileVariables)
+                                          .join(Ensemble)
+                                          .filter(Ensemble.name.in_(ensemble))
+                                          .filter(DataFile.index_time >= since).all())
+
+def Generate_Manifest(dsn, ensemble, since):
```

Comment:
In Python, one usually uses either CamelCaps (i.e. no underscore) or all_lower_case with underscores. Usage of capital letters is generally reserved for the names of classes, not functions. So I'd change this to `generate_manifest`.